3-extra/1-guess-the-number-project/script.js

```diff
@@ -1,30 +1,31 @@
 let randomNumber = Math.floor(Math.random() * 100 + 1);
 
 function guessNumber() {
-  //Collect input from the user
   let guess = document.querySelector(".inputs-Values").value;
-
-  //If the user inputs a bad input ie 0, empty string, number greater that 100, number less than zero Print "Please enter a number between 1 and 100"
-
-  //If the users guess is higher than the random number print Number is too high, try again (hint use .final-out class to print)
-
-  //If the users guess is lower than the random number print Number is too low, try again  (hint use .final-out class to print)
-
-  //If the user has guessed the random number correctly print out the randomNumber with a message "Guess is correct. You win!"
+  let output = document.querySelector(".final-output");
+  let tries = document.querySelector(".Tries-output");
+  if (!guess || guess > 100 || guess < 1) {
```

Comment:
nice covering of an edgecase with `!guess` - although as a warning the `!x` shorthand only checks if `x` is `falsey` rather than `null/undefined`. This means that `0, "", false` are all classed as `falsey`, so  if `0` is a valid input this would be a bug. https://developer.mozilla.org/en-US/docs/Glossary/Falsy